docs/src/api/params.md

```diff
@@ -303,28 +303,39 @@ Emulates consistent window screen size available inside web page via `window.scr
 
 Target URL.
 
-## fetch-option-params
+## js-python-fetch-option-params
```

Comment:
I'd rename this and similar back to `fetch-option-param` since it's the default style.

Reply:
They are now available only in js and python and for such cases we include languages into the prefix in other places, so this is consistent with existing approach.